src/compiler/diagnosticMessages.json

```diff
@@ -4026,5 +4026,61 @@
     "Add definite assignment assertion to property '{0}'": {
         "category": "Message",
         "code": 95020
+    },
+    "Fix all like: ": {
+        "category": "Message",
+        "code": 95021
+    },
+    "Add all missing members": {
+        "category": "Message",
+        "code": 95022
+    },
+    "Infer all types from usage": {
+        "category": "Message",
+        "code": 95023
+    },
+    "Delete all unused declarations": {
+        "category": "Message",
+        "code": 95024
+    },
+    "Prefix all unused declarations with '_'": {
+        "category": "Message",
+        "code": 95025
+    },
+    "Fix all spelling errors": {

```

Comment:
We might want to hedge here because we are definitely not going to fix all spelling errors. 😄